docs/operator-manual/cluster-management.md

```diff
@@ -0,0 +1,23 @@
+# Cluster Management
+
+This guide is for operators looking to manage clusters on the CLI. If you want to use Kubernetes resources for this, check out [Declarative Setup](./declarative-setup.md#clusters).
+
+Not all commands are described here, see the [argocd cluster Command Reference](../user-guide/commands/argocd_cluster.md) for all available commands.
+
+## Adding a cluster
+
+Run `argocd cluster add $CONTEXT`.
+
+If you're unsure about the context names, run `kubectl config get-contexts` to get them all listed.
+
+This will connect to the cluster and install the necessary resources for ArgoCD to connect to it.
+Note that you will need privileged access to the cluster.
+
+## Removing a cluster
+
+Run `argocd cluster rm $NAME`.
+
+This removes the cluster with the specified name.
+
+!!!note "in-cluster cannot be removed"
```

Comment:
I believe a space after !!! is needed.

Reply:
I checked with the other documentation files, none of them have a space between `!!!` and `note` (or `warning`).

Reply:
ok, good to know that.